docs/content/tutorials/tutorial-streams.md

```diff
@@ -0,0 +1,134 @@
+the [single-machine quickstart](quickstart.html) and have it running on your local machine. You 
+don't need to have loaded any data yet.
+
+Once that's complete, you can load your own dataset by writing a custom ingestion spec.
+
+## Writing an ingestion spec
+
+When loading streams into Druid, we recommend using the [stream push](../ingestion/stream-push.html) 
+process. In this tutorial we'll be using [Tranquility Server](../ingestion/stream-ingestion.html#server) to push 
+data into Druid over HTTP.
+
+```note-info
+This tutorial will show you how to push streams to Druid using HTTP, but Druid additionally supports 
+a wide variety of batch and streaming loading methods. See the *[Loading files](batch-ingestion.html)* 
+and *[Loading streams](stream-ingestion.html)* pages for more information about other options, 
+including from Hadoop, Kafka, Storm, Samza, Spark Streaming, and your own JVM apps.
+```
+
+You can prepare for loading a new dataset over HTTP by writing a custom Tranquility Server 
+configuration. The bundled configuration is in `conf-quickstart/tranquility/server.json`, which 
+you can modify for your own needs.
+
+The most important questions are:
+
+  * What should the dataset be called? This is the "dataSource" field of the "dataSchema".
+  * Which field should be treated as a timestamp? This belongs in the "column" of the "timestampSpec".
+  * Which fields should be treated as dimensions? This belongs in the "dimensions" of the "dimensionsSpec".
+  * Which fields should be treated as measures? This belongs in the "metricsSpec".
+
+Let's use a small JSON pageviews dataset as an example, with records like:
+
+```json
+{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "alice", "latencyMs": 32}
+```
+
+So the answers to the questions above are:
+
+  * Let's call the dataset "pageviews".
+  * The timestamp is the "time" field.
+  * Good choices for dimensions are the string fields "url" and "user".
+  * Good choices for measures are a count of pageviews, and the sum of "latencyMs". Collecting that 
+sum when we load the data will allow us to compute an average at query time as well.
+
+Now, edit the existing `conf-quickstart/tranquility/server.json` file by altering these 
+sections:
+
+  1. Change the key `"metrics"` under `"dataSources"` to `"pageviews"`
+  2. Alter these sections under the new `"pageviews"` key:
+  ```json
+  "dataSource": "pageviews"
+  ```
+
+  ```json
+  "timestampSpec": {
+       "format": "auto",
+       "column": "time"
+  }
+  ```
+
+  ```json
+  "dimensionsSpec": {
+       "dimensions": ["url", "user"]
+  }
+  ```
+
+  ```json
+  "metricsSpec": [
+       {"name": "views", "type": "count"},
+       {"name": "latencyMs", "type": "doubleSum", "fieldName": "latencyMs"}
+  ]
+  ```
+
+## Restarting the server
+
+Restart the server to pick up the new configuration file by CTRL-C out of Tranquility and starting it up again.
```

Comment:
This is just an awkward read.  One possibility is to say ... " by stopping Tranquiity (with CTRL-C) and starting it up again).